Name: Hide Sprint Navigation App
Version: 1.4.0-2
Author: v3ritas
Description: This patch will hide the Sprint Navigation App in the Launcher.  Thanks to opacityzero at PreCentral for updating the patch to 1.4.0.

--- .orig/usr/palm/applications/com.telenav.app.sprintnavigation/appinfo.json
+++ /usr/palm/applications/com.telenav.app.sprintnavigation/appinfo.json
@@ -13,6 +13,7 @@
     "splashicon":"icon-256x256.png",
     "data-directory":"file:///var/telenav/",
     "mockSystem": false,
+	"visible": "false"
     "mimeTypes": [{"scheme": "mapto"}, {"scheme": "maploc"}]
 }
 
